docs/literate/make.jl

```diff
@@ -0,0 +1,122 @@
+using Literate
+using Test
+using HTTP
+import Pkg
+
+# Function to create markdown and notebook files for specific file
+function create_files(title, file; folder="")
+    if folder == ""
+        notebook_filename = "$(splitext(file)[1]).ipynb"
+    else
+        notebook_filename = "$folder/$(splitext(file)[1]).ipynb"
+    end
```

Comment:
```suggestion
    notebook_filename = first(splitext(file)) * ".ipynb"
    if folder == ""
        notebook_filename = joinpath(folder, notebook_filename)
    end
```
This avoids writing the same logic twice.